docs/hugo/content/_index.md

```diff
@@ -105,7 +105,12 @@ recommended on AKS Free-tier clusters.
 See [CRD management](https://azure.github.io/azure-service-operator/guide/crd-management/) for more details.
 {{% /alert %}}
 
-   Alternatively you can install from the [release YAML directly](https://azure.github.io/azure-service-operator/guide/installing-from-yaml/).
+{{% alert title="Note" %}}
+When installing ASO in an AKS cluster, see [AKS Best Practices]({{< relref "best-practices#aks-best-practices" >}}).
+into an AKS cluster.
```

Comment:
Is this second line needed?

Reply:
No, I think holdover from previous phrasing. Good catch.